geti_sdk/geti.py

```diff
@@ -252,19 +252,26 @@ def credit_balance(self) -> Optional[int]:
         return balance.available if balance is not None else None
 
     def get_project(
-        self, project_name: str, project_id: Optional[str] = None
+        self,
+        project_name: Optional[str] = None,
+        project_id: Optional[str] = None,
+        project: Optional[Project] = None,
     ) -> Project:
         """
-        Return the Intel® Geti™ project named `project_name`, if any. If no project by
-        that name is found on the Intel® Geti™ server, this method will raise a
-        KeyError.
+        Return the Intel® Geti™ project by name or ID, if any.
+        If a project object is passed, the method will return the updated object.
+        If no project by that name is found on the Intel® Geti™ server,
+        this method will raise a KeyError.
 
         :param project_name: Name of the project to retrieve
+        :param project_id: ID of the project to retrieve. If not specified, the
+            project with name `project_name` will be retrieved.
+        :param project: Project object to update. If not specified, the project with
```

Comment:
Just looked at the `ProjectClient.get_project` method, the behaviour is clear now. Could you add a brief description of the parameter order here? Mainly because users may use the `Geti` object as the entrypoint to the API

Reply:
I updated the docstring, please see if the explanations are clearer now